### ISSUE TITLE
Supervisor is not privileged

### ISSUE THREAD

**Issue by matthew73210** [home-assistant/supervisor#2351]
### Describe the issue

Updated supervisor to 2020.12.6. Unable to update add-one or core.

The following errors appear and are not resolvable:

> You are running an unsupported installation. Learn more

> Your installtion is running in an unhealthy state. Learn more

If I click on "learn more":

> Supervisor is not privileged

> he Docker environment is not working properly

Ran the "convenience installation script" as mentioned in [Privileged](https://www.home-assistant.io/more-info/unsupported/privileged), rebooted but the problem still persists.

Base OS is up to date.

I think that i've added all that I can find. If missing anything just ask and i'll do my best to find it.

### Steps to reproduce

1. Update supervisor

### Enviroment details

- **Operating System:**: Debian GNU/Linux 10 (buster)
- **Supervisor version:**: 2020.12.6
- **Home Assistant version**: 0.116.4

### Supervisor logs

<details>
<summary>Supervisor logs</summary>

20-12-07 06:39:17 INFO (MainThread) [supervisor.host.services] Updating service information
20-12-07 06:39:17 INFO (MainThread) [supervisor.host.network] Updating local network information
20-12-07 06:39:19 INFO (MainThread) [supervisor.host.sound] Updating PulseAudio information
20-12-07 06:39:19 INFO (MainThread) [supervisor.host] Host information reload completed

20-12-07 07:38:56 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.RUNNING
20-12-07 07:38:56 INFO (MainThread) [supervisor.resolution.check] System checks complete
20-12-07 07:38:56 INFO (MainThread) [supervisor.snapshots] Found 4 snapshot files
20-12-07 07:38:58 INFO (MainThread) [supervisor.updater] Fetching update data from https://version.home-assistant.io/stable.json

20-12-07 07:40:58 INFO (MainThread) [supervisor.misc.scheduler] Shutting down scheduled tasks
20-12-07 07:40:58 INFO (MainThread) [supervisor.api] Stopping API on 172.30.32.2
20-12-07 07:40:58 INFO (MainThread) [supervisor.misc.hwmon] Stopped Supervisor hardware monitor
20-12-07 07:40:58 INFO (MainThread) [supervisor.core] Supervisor is down - 100
20-12-07 07:40:58 INFO (MainThread) [__main__] Closing Supervisor
20-12-07 07:41:02 INFO (MainThread) [supervisor.bootstrap] Use the old homeassistant repository for machine extraction
20-12-07 07:41:02 INFO (MainThread) [__main__] Initializing Supervisor setup
20-12-07 07:41:02 INFO (MainThread) [supervisor.bootstrap] Initializing Supervisor Sentry
20-12-07 07:41:02 INFO (MainThread) [supervisor.bootstrap] Seting up coresys for machine: qemux86-64
20-12-07 07:41:02 INFO (SyncWorker_0) [supervisor.docker.supervisor] Attaching to Supervisor homeassistant/amd64-hassio-supervisor with version 2020.12.6
20-12-07 07:41:03 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.INITIALIZE
20-12-07 07:41:03 WARNING (MainThread) [supervisor.resolution.evaluations.base] Supervisor does not run in Privileged mode. (more-info: https://www.home-assistant.io/more-info/unsupported/privileged)
20-12-07 07:41:03 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
20-12-07 07:41:03 INFO (MainThread) [__main__] Setting up Supervisor
20-12-07 07:41:03 INFO (MainThread) [supervisor.api] Starting API on 172.30.32.2
20-12-07 07:41:10 WARNING (MainThread) [supervisor.dbus.rauc] Host has no rauc support. OTA updates have been disabled.
20-12-07 07:41:10 INFO (MainThread) [supervisor.host.info] Updating local host information
20-12-07 07:41:10 INFO (MainThread) [supervisor.host.services] Updating service information
20-12-07 07:41:10 INFO (MainThread) [supervisor.host.network] Updating local network information
20-12-07 07:41:12 INFO (MainThread) [supervisor.host.sound] Updating PulseAudio information
20-12-07 07:41:12 INFO (MainThread) [supervisor.host] Host information reload completed
20-12-07 07:41:12 INFO (MainThread) [supervisor.host.apparmor] Loading AppArmor Profiles: {'hassio-supervisor'}
20-12-07 07:41:12 INFO (MainThread) [supervisor.host.services] Reloading local service hassio-apparmor.service
20-12-07 07:41:12 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to homeassistant/amd64-hassio-dns with version 2020.11.0
20-12-07 07:41:12 INFO (MainThread) [supervisor.plugins.dns] Updated /etc/resolv.conf
20-12-07 07:41:12 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to homeassistant/amd64-hassio-audio with version 17
20-12-07 07:41:12 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to homeassistant/amd64-hassio-cli with version 2020.11.1
20-12-07 07:41:12 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to homeassistant/amd64-hassio-observer with version 2020.10.1
20-12-07 07:41:12 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to homeassistant/amd64-hassio-multicast with version 3
20-12-07 07:41:12 INFO (MainThread) [supervisor.updater] Fetching update data from https://version.home-assistant.io/stable.json
20-12-07 07:41:13 INFO (SyncWorker_1) [supervisor.docker.interface] Attaching to homeassistant/qemux86-64-homeassistant with version 0.116.4
20-12-07 07:41:13 INFO (MainThread) [supervisor.homeassistant.secrets] Loaded 1 Home Assistant secrets
20-12-07 07:41:13 INFO (MainThread) [supervisor.hassos] No Home Assistant Operating System found
20-12-07 07:41:13 INFO (MainThread) [supervisor.store.git] Loading add-on /data/addons/git/a0d7b954 repository
20-12-07 07:41:13 INFO (MainThread) [supervisor.store.git] Loading add-on /data/addons/core repository
20-12-07 07:41:19 INFO (MainThread) [supervisor.store] Loading add-ons from store: 66 all - 66 new - 0 remove
20-12-07 07:41:19 INFO (MainThread) [supervisor.addons] Found 11 installed add-ons
20-12-07 07:41:19 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to hassioaddons/zerotier-amd64 with version 0.7.3
20-12-07 07:41:19 INFO (SyncWorker_2) [supervisor.docker.interface] Attaching to homeassistant/amd64-addon-mosquitto with version 5.1
20-12-07 07:41:19 INFO (SyncWorker_3) [supervisor.docker.interface] Attaching to hassioaddons/portainer-amd64 with version 1.2.2
20-12-07 07:41:19 INFO (SyncWorker_1) [supervisor.docker.interface] Attaching to homeassistant/amd64-addon-configurator with version 5.1.0
20-12-07 07:41:19 INFO (SyncWorker_2) [supervisor.docker.interface] Attaching to hassioaddons/unifi-amd64 with version 0.19.2
20-12-07 07:41:19 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to homeassistant/amd64-addon-check_config with version 3.6.0
20-12-07 07:41:19 INFO (SyncWorker_4) [supervisor.docker.interface] Attaching to hassioaddons/glances-amd64 with version 0.9.0
20-12-07 07:41:19 INFO (SyncWorker_5) [supervisor.docker.interface] Attaching to esphome/esphome-hassio-amd64 with version 1.15.3
20-12-07 07:41:19 INFO (SyncWorker_2) [supervisor.docker.interface] Attaching to homeassistant/amd64-addon-duckdns with version 1.12.1
20-12-07 07:41:19 INFO (SyncWorker_1) [supervisor.docker.interface] Attaching to hassioaddons/ssh-amd64 with version 7.6.0
20-12-07 07:41:19 INFO (SyncWorker_3) [supervisor.docker.interface] Attaching to hassioaddons/log-viewer-amd64 with version 0.9.0
20-12-07 07:41:20 INFO (MainThread) [supervisor.snapshots] Found 4 snapshot files
20-12-07 07:41:20 INFO (MainThread) [supervisor.discovery] Loaded 1 messages
20-12-07 07:41:20 INFO (MainThread) [supervisor.ingress] Loaded 1 ingress sessions
20-12-07 07:41:20 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.SETUP
20-12-07 07:41:20 INFO (MainThread) [supervisor.resolution.check] System checks complete
20-12-07 07:41:20 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.SETUP
20-12-07 07:41:20 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
20-12-07 07:41:20 INFO (MainThread) [__main__] Running Supervisor
20-12-07 07:41:20 WARNING (MainThread) [supervisor.core] System running in a unsupported environment!
20-12-07 07:41:20 CRITICAL (MainThread) [supervisor.core] System running in a unhealthy state and need manual intervention!
20-12-07 07:41:20 INFO (MainThread) [supervisor.addons] Phase 'AddonStartup.INITIALIZE' starting 0 add-ons
20-12-07 07:41:20 INFO (MainThread) [supervisor.core] Supervisor reboot detected
20-12-07 07:41:20 INFO (MainThread) [supervisor.misc.tasks] All core tasks are scheduled
20-12-07 07:41:20 INFO (MainThread) [supervisor.misc.hwmon] Started Supervisor hardware monitor
20-12-07 07:41:20 INFO (MainThread) [supervisor.core] Supervisor is up and running
20-12-07 07:41:20 INFO (MainThread) [supervisor.host.info] Updating local host information
20-12-07 07:41:20 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state CoreState.RUNNING
20-12-07 07:41:20 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
20-12-07 07:41:20 INFO (MainThread) [supervisor.host.services] Updating service information
20-12-07 07:41:20 INFO (MainThread) [supervisor.host.network] Updating local network information
20-12-07 07:41:21 INFO (MainThread) [supervisor.host] Host information reload completed

20-12-07 07:48:18 WARNING (MainThread) [supervisor.addons.validate] Unknown options web
20-12-07 07:48:18 WARNING (MainThread) [supervisor.addons.validate] Unknown options web
20-12-07 07:48:18 WARNING (SyncWorker_1) [supervisor.docker.addon] SSH & Web Terminal running with disabled protected mode!
20-12-07 07:48:18 INFO (SyncWorker_1) [supervisor.docker.interface] Cleaning addon_a0d7b954_ssh application
20-12-07 07:48:21 INFO (SyncWorker_1) [supervisor.docker.addon] Starting Docker add-on hassioaddons/ssh-amd64 with version 7.6.0
20-12-07 07:48:24 INFO (MainThread) [supervisor.api.security] /host/info access from a0d7b954_ssh
20-12-07 07:49:12 INFO (MainThread) [supervisor.api.security] /cli/info access from a0d7b954_ssh

</details>

**Comment by Jason-nz** [comment]
https://github.com/home-assistant/supervised-installer/pull/135

**Comment by Jason-nz** [comment]
Also duplicate of https://github.com/home-assistant/supervisor/issues/2343

**Comment by matthew73210** [comment]
> Also duplicate of #2343

Seen that issue, I was asked to open a new one as I'm on a "supported" system. Thanks for your swift reply 👍

**Comment by Jason-nz** [comment]
Ah yes 👍

**Comment by Jason-nz** [comment]
Does making the change to /sbin/hassio-supervisor in the linked pull request fix your issue?

After  editing that file run:
systemctl restart hassio-supervisor | journalctl -fu hassio-supervisor

**Comment by pvizeli** [comment]
You need to run the installer on a fresh system or remove all containers and rerun the installer script which needs to create a new Supervisor container. After that, it works on Debian.

A short reminder, To use the Supervisord, means you are self-responsible for OS. If that is somethings out of your Scope, the OS doing an Amazing job for you :)

**Comment by pvizeli** [comment]
@Jason-nz Ubuntu is an Unsupported system. Please ask for help on: https://community.home-assistant.io/t/installing-home-assistant-supervised-on-ubuntu-18-04-4/200020

**Comment by Jason-nz** [comment]
@matthew73210
The installer script has just been updated. Try running that again now and see if it solves your issue. 🤞

Seems to be working for me on U****u 🤫

**Comment by matthew73210** [comment]
Sorry for the late reply.

All seems to work now. What I did was remove all the add-ons, run the script. Reboot and the problem still persisted. I ventured into dock "docker ps" and noticed that the "hassio_supervisor" hadn't been reset and was still running since boot. Even if the script had been run.

Then I thought, maybe as the docker container is running, and restarts as soon as you stop it. Perhaps the script isn't actually replacing it. So I forced docker to remove the container "docker container rm -f hassio_supervisor". Ran the installer.sh and bingo. No more complaints from the supervisor. Could this be intgrated into the script? Force removal of old hassio_supervisor container?

I can now update everything too.

Should I close this issue ?
